### DISCUSSION TITLE
[Question] Anyone know how to force a lg view on mobile?

### DISCUSSION THREAD

**Discussion by veggie2u** [tailwindlabs/tailwindcss#9699] [Help] [answered]
I have media queries set up to show cards or tables depending on the size of the screen. Is there a way I can let the user choose they want to see tables on mobile? I was trying to figure out a way short of modifying every media query with an if in js.

For example, I have a theme switcher that sets 
`document.documentElement.setAttribute('data-theme', newTheme)`

I want to have some way to let the user set whether they see mobile or desktop.

**Comment by wongjn** [comment, marked as answer]
You could use a class on a container which you could toggle:
```html
<body class="see-tables">
  <!-- … -->
  <div class="lg:text-red-500 [.see-tables_&]:text-red-500">
    <!-- … -->
  </div>
  <!-- … -->
</body>
```
Here, the div will have red text on the `lg` screen **or** when `see-tables` has been toggled. If you will be using this a lot, you could reduce this utility duplication by abstracting it to a variant via a plugin.

**Reply by veggie2u** [reply]
What is the [.see-tables_&]: doing? Is that working like the md: and lg: do? And what is the _& part?

**Reply by wongjn** [reply]
`[.see-tables_&]` is an [arbitrary variant](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants). The underscore (`_`) is a substitute for a space. The ampersand (`&`) represents the selector being modified, or "this element". After these two concepts, it is a general CSS selector, `.see-tables <element>`, and thus will only match when an ancestor has the `see-tables` class, just like what `.see-tables element` as a CSS selector would match.

**Comment by Amyconnell** [comment]
Ensure that your mobile device and the TV are connected to the same Wi-Fi. From your phone, go to SETTINGS, then select SHARE AND CONNECT. Under SCREEN SHARE category, select SCREEN SHARING or MIRROR SCREEN.   https://www.imyccpay.com
